Tests/StreamChatTests/Controllers/ChannelListController/ChannelListController_Tests.swift

```diff
@@ -1619,6 +1619,63 @@ final class ChannelListController_Tests: XCTestCase {
         )
     }
 
+    func test_filterPredicate_autocompleteInCollection_returnsExpectedResults() throws {
+        let cid = ChannelId.unique
+
+        try assertFilterPredicate(
+            .autocomplete(.memberName, text: "test"),
+            channelsInDB: [
+                .dummy(channel: .dummy(cid: .unique, members: [
+                    .dummy(user: .dummy(userId: .unique, name: "userA")),
+                    .dummy(user: .dummy(userId: .unique, name: "userC"))
+                ])),
+                .dummy(channel: .dummy(cid: .unique, members: [
+                    .dummy(user: .dummy(userId: .unique, name: "userB"))
+                ])),
+                .dummy(channel: .dummy(cid: cid, members: [
+                    .dummy(user: .dummy(userId: .unique, name: "test"))
+                ]))
+            ],
+            expectedResult: [cid]
+        )
+    }
+
+    func test_filterPredicate_autocompleteInNonCollection_returnsExpectedResults() throws {
+        let cid = ChannelId.unique
+
+        try assertFilterPredicate(
+            .autocomplete(.name, text: "test"),
+            channelsInDB: [
+                .dummy(channel: .dummy(cid: .unique, name: "channelA")),
+                .dummy(channel: .dummy(cid: .unique, name: "channelB")),
+                .dummy(channel: .dummy(cid: cid, name: "test"))
+            ],
+            expectedResult: [cid]
+        )
```

Comment:
In this test we are not testing the "autocomplete" feature, we need to verify if indeed it returns the channel with the correct "prefix"
```suggestion
        let cid = ChannelId.unique

        try assertFilterPredicate(
            .autocomplete(.name, text: "test"),
            channelsInDB: [
                .dummy(channel: .dummy(cid: .unique, name: "channelA")),
                .dummy(channel: .dummy(cid: .unique, name: "channelB")),
                .dummy(channel: .dummy(cid: cid, name: "testChannel"))
            ],
            expectedResult: [cid]
        )
```